### ISSUE TITLE
incompatibility issues

### ISSUE THREAD

**Issue by MMetty10024** [Feodor2/Mypal#342]
thanks for the great community support , could you please provide a compatibility solution for my banking website (scotiaonline.com) , I'm using version 29.0.1 , many thanks.

**Comment by Feodor2** [comment]
Did you try useragent override, look https://github.com/Feodor2/Mypal/issues/339#issuecomment-782442741

**Comment by MMetty10024** [comment]
yes I tried that before with no luck , but with this new 78.0 version I can access the site but still receive the incompatibility issue which is fine as previously I couldn't access it at all , thanks for the quick response , your words have power.
